tmpl/class.tmpl

```diff
@@ -173,10 +173,12 @@
     {{/if}}
 
     {{#if cls._class}}
+    {{#unless cls.hideconstructor}}
     <h2>Constructor</h2>
     <!-- Constructor -->
     {{> method obj=cls}}
-    {{/if}}
+    {{/unless}}
+    {{/if}}`
```

Comment:
Did you mean to include the back tick here?

Reply:
Oh snap, sorry that was a typo